DOCS/man/options.rst

```diff
@@ -6573,17 +6573,22 @@ them.
         Blend the frame against the background color (``--background``, normally
         black).
     yes
-        Try to create a framebuffer with alpha component. This only makes sense
-        if the video contains alpha information (which is extremely rare) or if
-        you make the background color transparent. May not be supported on all
-        platforms. If alpha framebuffers are unavailable, it silently falls
-        back on a normal framebuffer. Note that if you set the ``--fbo-format``
-        option to a non-default value, a format with alpha must be specified,
-        or this won't work. Whether this really works depends on the windowing
-        system and desktop environment.
+        Render the frame as-is, preserving its alpha component. Blending is
+        done on the mpv window by the windowing system. This only works
+        if ``--window-transparent`` is enabled.
```

Comment:
Should `--alpha=yes` also automatically imply `--window-transparent=yes`? Maybe `--window-transparent=auto` should be the default, which means `yes` iff `--alpha=yes` (and the file has an alpha channel).

Reply:
I tried to do this, but the part "and the file has an alpha channel" is more difficult than anticipated. The GL context is created before init_video() is called, and so before we have knowledge of whether the file has an alpha channel.

Even if that were not the case, we can easily imagine a case where an image without transparency is loaded, and later another one that has transparency. So the GL context would have to be recreated when that happens. 
According to @Dudemanguy this is difficult-to-not-doable
>I would only forsee problems trying to recreate contexts like this so maybe the auto mode is just a no-go
if you make a context disappear while a vo thinks it's still up and running... probably just bad things will happen

As a simpler alternative, we could still make a `window-transparent=auto` mode which only depends on the alpha value, and would be yes if `alpha=yes`, and no otherwise? This also has the advantage of not breaking the current behavior of setting `alpha=yes` by itself.

Reply:
> As a simpler alternative, we could still make a `window-transparent=auto` mode which only depends on the alpha value, and would be yes if `alpha=yes`, and no otherwise? This also has the advantage of not breaking the current behavior of setting `alpha=yes` by itself.

I think I would prefer this approach.

Reply:
This is now the case. I've also dropped the warning message since it would only matter when the user explicitly sets `alpha=yes` and `window-transparent=no` which does not make much sense. I also was not able to test with gpu-next, but since the change is very similar to gpu, I'm somewhat confident.